packages/code/src/CopyButton/CopyButton.spec.tsx

```diff
@@ -51,11 +58,11 @@ describe('CopyButton', () => {
   test(`tooltip displays "${COPIED_TEXT}" text for ${COPIED_SUCCESS_DURATION}ms after trigger is clicked`, async () => {
     const { getByTestId, queryByTestId } = renderCopyButton({});
     const copyButton = getByTestId(testIds.copyButton);
-    let tooltip = queryByTestId(testIds.tooltip);
 
+    userEvent.click(copyButton);
+
+    let tooltip = queryByTestId(testIds.tooltip);
```

Comment:
I'm not 100% certain, but if you wanted to shorten this even more, i think you can get away with
```
let tooltip = await findByTestId(testIds.tooltip);
expect(tooltip).toHaveTextContent(COPIED_TEXT);
```

Reply:
it doesn't complete the re-render with the text update before the expect statement, so I have to keep the `waitFor` wrapping the expect statement